api/v1alpha1/datamigrate_types.go

```diff
@@ -88,6 +88,21 @@ type DataMigrateSpec struct {
 	// Resources that will be requested by the DataMigrate job. <br>
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// Parallelism defines the parallelism tasks
+	// +optional
+	// +kubebuilder:default:=1
+	// +kubebuilder:validation:Minimum=1
+	Parallelism int32 `json:"parallelism,omitempty"`
+
+	// SSHSecretName defines the secret name used for ssh. It must be set when Parallelism is greater than 1.
+	// +optional
+	SSHSecretName string `json:"sshSecretName,omitempty"`
+
+	// WorkersReadyTimeout defines timeout before parallel workers ready
+	// +optional
+	// +kubebuilder:default:=15
+	WorkersReadyTimeout int32 `json:"workers_ready_timeout,omitempty"`
```

Comment:
I suggest using workersReadyTimeoutSeconds instead of workers_ready_timeout  which is better fit for Kubernetes naming conventions.

```yaml
WorkersReadyTimeoutSeconds int32 `json:"workersReadyTimeoutSeconds,omitempty"`
```

Reply:
From my understanding, I prefer to make `WorkersReadyTimeout` as an option in `DataMigrate.spec.options`. Making it a field might be too heavy.

Reply:
Besides, I think the default timeout seconds now may be too short in some cases. For example, the launcher job's pod starts but the statefulset's pods may be loading image. How about changing the default timeout seconds to 180 seconds?

Reply:
fixed.

Reply:
fixed, crd use a map to define related configurations. default timeout is 180 seconds now.